js/data.js

```diff
@@ -214,7 +214,17 @@ var data = {
         kennung: [IST, OSIS, SAMT],
```

Comment:
We should probably name this `"Algorithmix<br />WS15/16"`, otherwise it gets confusing why there are two `Algorithmix`.